src/API/Site/Controllers/RestAPI/AuthController.php

```diff
@@ -62,6 +70,17 @@ public function register_routes() {
 				],
 			]
 		);
+
+		$this->register_route(
+			'rest-api/authorize',
+			[
+				[
+					'methods'             => TransportMethods::DELETABLE,
+					'callback'            => $this->delete_authorize_callback(),
+					'permission_callback' => $this->get_permission_callback(),
+				],
+			]
+		);
```

Comment:
I think we can move the DELETE method under the GET method of the endpoint above, like what Google account controller did: https://github.com/woocommerce/google-listings-and-ads/blob/8ebe1ecaf902ea114ae426d2abd86aebc960b560/src/API/Site/Controllers/Google/AccountController.php#L58-L68